### DISCUSSION TITLE
Changing github handle or username

### DISCUSSION THREAD

**Discussion by akibhaider** [community/community#133913] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can I change my github handle name? How frequently can it be done? Is there any restriction?

**Comment by hamfan524** [comment]
I'm not sure if I understand the question well, but let me explain.

1.	Log in to GitHub: Go to [GitHub](https://github.com/) and log in to your account.
2.	Go to Settings: Click on your profile picture in the upper-right corner, and select “Settings” from the dropdown menu.
3.	Account Settings: In the left-hand sidebar, click on “Account”.
4.	Change Username: Under the “Change username” section, enter your new desired username. If the username is available, you’ll see a green checkmark next to it.
5.	Confirm Changes: Click “Change username”. GitHub will confirm the change and update your username.

Frequency and Restrictions:
- Frequency: There is no strict limit on how often you can change your username. 
- Restrictions:
    - The new username must be unique and not currently in use by another account.
    - The username can only contain alphanumeric characters and hyphens (-), cannot have consecutive hyphens, cannot begin or end with a hyphen, and must be 1-39 characters in length.
    - Some usernames may be restricted due to GitHub’s reserved names policy (e.g., admin, support, etc.).
